### ISSUE TITLE
GLib error occurs in text mode

### ISSUE THREAD

**Issue by bdbell** [veracrypt/VeraCrypt#134]
Installed veracrypt from Unit193 PPA archive, version 1.19.  Running environment is Ubuntu 16.04 on x68_64, running at user level / not root.

When running commands to create and mount volumes in text mode, the following error appears when the command is complete.  It does not appear to affect the execution of the command.

GLib-GObject-CRITICAL **: g_type_class_unref: assertion 'g_class != NULL' failed

This error does not appear for commands that would not otherwise generate a GUI window, such as the '--dismount' command, or commands that are otherwise allowed to open a GUI window (where the '--text' option is omitted).

**Comment by Akrij** [comment]
Ich can confirm the issue as described by @bdbell for veracrypt **1.2.1** on a headless Ubuntu Server 16.04 (64 bit), installed from [unit193 PPA](https://launchpad.net/~unit193/+archive/ubuntu/encryption).

**Comment by piaseckim** [comment]
The same for me, error is shown every time `--text` parameter is given:
```
~$ veracrypt --text --version
VeraCrypt 1.21

(process:4172): GLib-GObject-CRITICAL **: g_type_class_unref: assertion 'g_class != NULL' failed
```

**Comment by trainmeditations** [comment]
I'm having a similar problem on Arch. Interestingly it seemed to not occur when I was trying to redirect stderr to stdout to capture the output. I ended up saving the terminal output and uploading that to a gist https://gist.github.com/trainmeditations/4ca2ac695f06c9a0ecc2f76485196704

**Comment by ElMostafaIdrassi** [comment]
Old issue but still relevant. This is a wxWidgets issue. It occurs when a program that uses wxWidgets can also run using just CLI, which is the case for VeraCrypt. When exiting, the program shows that error. 

This has been fixed apparently in version 3.0.3. So, if your VeraCrypt makes use of a version <= 3.0.2 (i.e. linked dynamically against Ubuntu 16.04 wxWidgets), you may hit this issue.

See [this](https://trac.wxwidgets.org/ticket/17253), [this ](https://github.com/wxWidgets/wxWidgets/commit/70faad3429eee7bc2be4d457b490af7c4899c58d )and [this](https://sourceforge.net/p/veracrypt/tickets/26/)
